src/app/app-routing.module.ts

```diff
@@ -1,21 +1,29 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { PerfilUsuarioComponent } from './perfil-usuario/perfil-usuario.component';
 import { HomeComponent } from './home/home.component';
 import { LoginComponent } from './acesso/login/login.component';
+import { CadastroComponent } from './acesso/cadastro/cadastro.component';
+import { CadastroClienteComponent } from './acesso/cadastro/cadastro-cliente/cadastro-cliente.component';
+import { CadastroPrestadorComponent } from './acesso/cadastro/cadastro-prestador/cadastro-prestador.component';
 import { AvaliacaoComponent } from './avaliacao/avaliacao.component'
 import { ListaServicosDetalhadosComponent } from './lista-servicos-detalhados/lista-servicos-detalhados.component';
+import { PerfilComponent } from './perfil/perfil.component';
+import { PrestadorComponent } from './prestador/prestador.component';
 
 const routes: Routes = [
-  {path: 'perfil', component: PerfilUsuarioComponent},
+  {path: 'perfil', component: PerfilComponent},
   {path: 'login', component: LoginComponent},
+  {path: 'cadastro', component: CadastroComponent},  
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6332072)